README.md

```diff
@@ -107,15 +106,45 @@ $ docker-compose up platformdata # add the '-d' flag to daemonize and run in bac
 **NOTE**: In order for the above to work, you will need to set `COGNITO_ADMIN_PASSWORD`, `AWS_ACCESS_KEY_ID`, and `AWS_SECRET_ACCESS_KEY` in a file named `.env` in the sinopia_editor root.
 
 ### Building latest Docker Image
+
 Before building the latest Docker Image, run `npm run build` to update the `dist` folder with the current build.
-To build the latest version of the [Sinopia Editor][GIT_REPO], you
-can build with the
-`docker build -t ld4p/sinopia_editor --no-cache=true .` command.
+To build the latest version of the [Sinopia Editor][GIT_REPO], you can build with the `docker build -t ld4p/sinopia_editor:latest --no-cache=true .` command. **NOTE** that images tagged with `latest` will not be deployed to any of our AWS environments. See below for how to build and deploy images
+
+### Pushing Docker Image to DockerHub
 
-### Pushing Docker Image to Dockerhub
 Run `docker login` and enter the correct credentials to your docker account.
 Once successfully authenticated, run `docker push ld4p/sinopia_editor:latest`.
-Ask a member on the DevOps team to go into the AWS console to udpate https://sinopia.io
+Ask a member on the DevOps team to go into the AWS console to update https://sinopia.io
+
+### Updating Docker Image in AWS Dev Environment
+
+This section assumes you've already authenticated to DockerHub via `docker login` in the previous section, and also assumes you've run through the [AWS development environment setup](https://github.com/sul-dlss/terraform-aws/wiki/AWS-DLSS-Dev-Env-Setup) documentation and configured the AWS CLI.
+
+First, build a new `sinopia_editor` image tagged with `dev`. In order to do this, you **MUST** provide the dev-specific build args:
+
+```shell
+$ docker build -t ld4p/sinopia_editor:dev --build-arg SPOOF_SINOPIA_SERVER=false --build-arg TRELLIS_BASE_URL=https://trellis.development.sinopia.io --build-arg DEFAULT_PROFILE_SCHEMA_VERSION=0.0.2 --build-arg SINOPIA_GROUP=ld4p --build-arg SINOPIA_URI=https://development.sinopia.io --build-arg AWS_COGNITO_DOMAIN=https://sinopia-development.auth.us-west-2.amazoncognito.com --build-arg COGNITO_CLIENT_ID=2u6s7pqkc1grq1qs464fsi82at .
+```
+
+Then push the `dev`-tagged issue to DockerHub:
+
+```shell
+$ docker push ld4p/sinopia_editor:dev
+```
+
+Next, set an environment variable to the name of the DLSS AWS dev profile as described in the documentation above:
+
+```shell
+$ export AWS_PROFILE=development
+```
+
+And, finally, run the following `aws` CLI commands to refresh the dev ECS instance that runs the editor:
```

Comment:
At first I thought I had to prepend `aws` to all of these commands but then noticed that the first two were just setting variables. Maybe reword as "finally, set the task_arn and cluster_arn variables, and then run the following `aws` CLI command to refresh...."

Reply:
Done!